Client/Phpredis/ClientCluster.php

```diff
@@ -0,0 +1,1532 @@
+<?php
+
+namespace Snc\RedisBundle\Client\Phpredis;
+
+use RedisCluster;
+use Snc\RedisBundle\Logger\RedisLogger;
+
+/**
+ * PHP RedisCluster client with logger.
+ *
+ * @author Igor Scabini <furester@gmail.com>
+ */
+class ClientCluster extends RedisCluster
+{
+    /**
+     * @var RedisLogger
+     */
+    protected $logger;
+
+    /**
+     * @var string
+     */
+    protected $alias;
+
+    /**
+     * Constructor.
+     *
+     * @param array       $parameters List of parameters (only `alias` key is handled)
+     * @param RedisLogger $logger     A RedisLogger instance
+     */
+    public function __construct(array $seeds = array(), array $parameters = array(), RedisLogger $logger = null)
+    {
+        $this->logger = $logger;
+        $this->alias = isset($parameters['alias']) ? $parameters['alias'] : '';
```

Comment:
Null coalesce operator can be used `$this->alias = $parameters['alias'] ?? ''`

Reply:
fix